DTIPrep/DTIPrepRegister.pl

```diff
@@ -1009,21 +1155,24 @@ sub getPipelineDate {
 }
 
 
+=pod
 
+=head3 insertReports($minc, $registeredXMLFile, $registeredQCReportFile)
 
+Inserts in the MINC header the path to DTIPrep's QC text and XML reports and XML
+protocol.
+
+INPUT: minc file to modify header; path to the registered DTIPrep's XML
```

Comment:
MINC file for which the header should be modified; ...

Reply:
done